src/sql.rs

```diff
@@ -330,7 +341,7 @@ fn open(
         .with_init(|c| c.execute_batch("PRAGMA secure_delete=on;"));
     let pool = r2d2::Pool::builder()
         .min_idle(Some(2))
-        .max_size(4)
+        .max_size(10)
```

Comment:
I'm not sure you wanted to do this one.  now there will be more sql threads than there are job threads wanting to execute sql?  seems wasteful

Reply:
seems i was merging too fast :)

to max_size()  are these really threads? i thought these are sqlite database handles created as needed. but anyway - 4 seems to be too few - there are 4 worker threads, yes, but there is also the main thread and the ui also easily spans another thread and calls into rust-core.

Reply:
nb: source is here: https://github.com/sfackler/r2d2/blob/master/src/lib.rs#L436 - looks to me as if add_connection() creates the handle in a way but only as needed. not sure if any threads are created, though.

Reply:
ah, you're probably right that they are done on-demand only.  so i guess that helps a bit.  i did think r2d2 basically created one thread per connection.

I don't think this is enough worry to do something about right away, let's see how this behaves on the clients in practice.

Reply:
yip, we can tweak later. thanks for the review - and sorry for merging that fast :)